### PR TITLE
Handle multi-layer gimp xcf files.

### PR DESCRIPTION
**Pull request by mooflu**

### CODE REVIEW

**Comment by xupefei** [comment]
LGTM! Does this issue happen also to Photoshop PSD files?

File: QuickLook.Plugin/QuickLook.Plugin.ImageViewer/AnimatedImage/Providers/ImageMagickProvider.cs [hunk 1]
```diff
@@ -92,8 +92,19 @@ public override Task<BitmapSource> GetRenderedFrame(int index)
 
                 try
                 {
-                    using (var mi = new MagickImage(Path.LocalPath, settings))
+                    using (MagickImageCollection layers = new MagickImageCollection(Path.LocalPath, settings))
                     {
+                        IMagickImage<byte> mi;
+                        // Only flatten multi-layer gimp xcf files.
+                        if (Path.LocalPath.EndsWith(".xcf") && layers.Count > 1)
```
Comment (xupefei) [inline QuickLook.Plugin/QuickLook.Plugin.ImageViewer/AnimatedImage/Providers/ImageMagickProvider.cs:99]: @mooflu Ouch I shouldn't merge the PR so quickly -- this `LocalPath` should be lowercase before checking the `EndsWith`. Could you create a separate PR?

Comment (mooflu) [reply QuickLook.Plugin/QuickLook.Plugin.ImageViewer/AnimatedImage/Providers/ImageMagickProvider.cs:99]: Oops, will do.

**Comment by mooflu** [comment]
I believe a PSD file has an extra (first) layer that contains the flatted contents.

**Pull request merged**